ground/gcs/src/plugins/config/configplugin.cpp

```diff
@@ -139,7 +139,7 @@ void ConfigPlugin::onAutopilotDisconnect()
   */
 void ConfigPlugin::eraseAllSettings()
 {
-    QMessageBox msgBox;
+    QMessageBox msgBox((QWidget *)Core::ICore::instance()->mainWindow());
```

Comment:
Discussed use of c-casts on IRC already. AFAICT these could all be static_cast since we can guarantee mainWindow returns the right type? Otherwise dynamic_cast.

Reply:
If you _really_ don't want to do it. I guess I will clean them up later.